ProjFS.Mac/PrjFS.xcodeproj/project.pbxproj

```diff
@@ -40,7 +40,7 @@
 		4391F8B621E42AC50008103C /* PrjFSKext.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 4391F89C21E42AC40008103C /* PrjFSKext.cpp */; };
 		4391F8B721E42AC50008103C /* PrjFSClasses.hpp in Headers */ = {isa = PBXBuildFile; fileRef = 4391F89D21E42AC50008103C /* PrjFSClasses.hpp */; };
 		4391F8B821E42AC50008103C /* PrjFSLogUserClient.hpp in Headers */ = {isa = PBXBuildFile; fileRef = 4391F89E21E42AC50008103C /* PrjFSLogUserClient.hpp */; };
-		4391F8B921E42AC50008103C /* PrjFSProviderUserClient.hpp in Headers */ = {isa = PBXBuildFile; fileRef = 4391F89F21E42AC50008103C /* PrjFSProviderUserClient.hpp */; };
+		4391F8B921E42AC50008103C /* PrjFSProviderUserClientPrivate.hpp in Headers */ = {isa = PBXBuildFile; fileRef = 4391F89F21E42AC50008103C /* PrjFSProviderUserClientPrivate.hpp */; };
```

Comment:
haven't seen this pattern else where in `vfs4git`- what does Private specifier mean in a file name?

Reply:
You're right, we haven't used this elsewhere yet, [except in PR 500 which has yet to be merged](https://github.com/Microsoft/VFSForGit/pull/500#discussion_r256141241). In the `PrjFSProviderUserClientPrivate.hpp` case, it's purely to signify that the definitions there are not-API (opaque implementation detail) for most of the rest of the kext. For `VirtualizationRootsPrivate.hpp`, in addition to being not-API for the rest of the kext, it allows the unit tests to access a private struct definition in order to be able to set up specific test scenarios.

Reply:
After out-of-band discussion, it was decided we'll go ahead with the "…Private.hpp" approach, and have the private and testable headers side-by-side; testable will continue to contain the prototypes for functions that are private but need to be accessed from tests.